src/yunohost/tests/test_changeurl.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import time
+import requests
+
+from moulinette.core import init_authenticator
+from yunohost.app import app_install, app_changeurl, app_remove, app_map
```

Comment:
I got ImportError: No module named yunohost.app
I don't know why( maybe the mistake come from my yuno instance)